### DISCUSSION TITLE
Listening subscription not returning updates in web server running at localhost:8080

### DISCUSSION THREAD

**Discussion by blissnd** [99designs/gqlgen#1800] [Q&A]
### What happened?
The listening subscription returned no data when the updates occurred

### What did you expect?
New added data based on subscription (subscribed to) returns the new data on the right-hand panel at localhost:8080

### Minimal graphql.schema and models to reproduce
type Todo {
  id: ID!
  text: String!
  done: Boolean!
  user: User!
  textLength: Int!
}

type User {
  id: ID!
  name: String!
}

input NewTodo {
  text: String!
  userId: String!
}

type Query {
  todos: [Todo!]!
  users: [User!]!
}

input NewUser {
  name: String!
}

type Mutation {
  createTodo(input: NewTodo!): Todo!
  createUser(input: NewUser!): User!
}

type Subscription {
  userAdded: User!
}

[based on https://www.inovex.de/blog/graphql-application-golang-tutorial/]

### versions
 - `gqlgen version`? v0.13.0
 - `go version`? go version go1.16.2 linux/amd64
 - dep or go modules?

**Comment by blissnd** [comment]
![Screenshot from 2021-03-20 20-46-55](https://user-images.githubusercontent.com/11492266/111885094-88b01f00-89bd-11eb-9c79-a25e935a76ce.png)

**Comment by ghost** [comment]
Seeing your code for the subscription resolver would be useful.
